README.md

```diff
@@ -4,18 +4,30 @@ This code has been tested in [Mbed v6.8](https://os.mbed.com/docs/mbed-os/v6.8),
 
 ## Dependencies
 
-This component needs `colcon` and other Python 3 packages in order to build micro-ROS packages:
+On a non ARM machine you will need to install the ARM cross compiler toolchain:
 
 ```bash
-pip3 install catkin_pkg lark-parser empy colcon-common-extensions
+sudo apt install arm-none-eabi-g++
```

Comment:
```suggestion
sudo apt install gcc-arm-none-eabi
```

like in https://github.com/micro-ROS/micro_ros_mbed/blob/6209542e8738509dbef5f54f0c4bc61e6ad2ec45/.github/workflows/ci.yml#L34